exercises/zebra-puzzle/.meta/template.j2

```diff
@@ -0,0 +1,11 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        expected = "{{ case["expected"] }}"
+        self.assertEqual({{ case["property"] }}(), expected)
```

Comment:
You can fix the casing issue between the solution and the spec with the following syntax:
 ```
        self.assertEqual({{ case["property"] | to_snake }}(), expected)
```

Reply:
That is super convenient!